tests/UI/campaigns/functional/FO/05_contactUs/03_sendMessageAsAnonymous.js

```diff
@@ -0,0 +1,179 @@
+const helper = require('@utils/helpers');
+const loginCommon = require('@commonTests/BO/loginBO');
+const files = require('@utils/files');
+
+// Import pages
+// BO
+const dashboardPage = require('@pages/BO/dashboard');
+const customerServicePage = require('@pages/BO/customerService/customerService');
+
+// FO
+const foHomePage = require('@pages/FO/home');
+const foLoginPage = require('@pages/FO/login');
+const foContactUsPage = require('@pages/FO/contactUs');
+
+// Import test context
+const testContext = require('@utils/testContext');
+
+const baseContext = 'functional_FO_contactUs_sendMessageAsAnonymous';
+
+// Import data
+const ContactUsFakerData = require('@data/faker/contactUs');
+const {DefaultCustomer} = require('@data/demo/customer');
+
+const contactUsData = new ContactUsFakerData(
+  {
+    firstName: DefaultCustomer.firstName,
+    lastName: DefaultCustomer.lastName,
+    subject: 'Customer service',
+    emailAddress: DefaultCustomer.email,
+  },
+);
+
+let browserContext;
+let page;
+
+/*
+Go to FO
+Check if not connected
+Check errors
+Send a message on contact page
+Verify message on customer service page
+ */
+describe('FO - Contact us : Send message from contact us page with customer not logged', async () => {
+  // before and after functions
+  before(async function () {
+    browserContext = await helper.createBrowserContext(this.browser);
+    page = await helper.newTab(browserContext);
+
+    await files.createFile('.', `${contactUsData.fileName}.txt`, 'new filename');
+  });
+
+  after(async () => {
+    await helper.closeBrowserContext(browserContext);
+
+    await files.deleteFile(`${contactUsData.fileName}.txt`);
+  });
+
+  it('should open the shop page', async function () {
+    await testContext.addContextItem(this, 'testIdentifier', 'openShop', baseContext);
+
+    await foHomePage.goTo(page, global.FO.URL);
+
+    const isHomePage = await foHomePage.isHomePage(page);
+    await expect(isHomePage).to.be.true;
+  });
+
+  it('should check if that any account is connected', async function () {
+    await testContext.addContextItem(this, 'testIdentifier', 'checkIfCustomerNotConnected', baseContext);
+
+    const isCustomerConnected = await foHomePage.isCustomerConnected(page);
+    await expect(isCustomerConnected, 'Customer is connected!').to.be.false;
+  });
+
+  it('should go on contact us page', async function () {
+    await testContext.addContextItem(this, 'testIdentifier', 'goOnContactPage', baseContext);
+
+    // Go to contact us page
+    await foLoginPage.goToFooterLink(page, 'Contact us');
+
+    const pageTitle = await foContactUsPage.getPageTitle(page);
+    await expect(pageTitle).to.equal(foContactUsPage.pageTitle);
+  });
+
+  it('should check if the email is empty', async function () {
+    await testContext.addContextItem(this, 'testIdentifier', 'checkEmptyEmail', baseContext);
+
+    await foContactUsPage.selectByVisibleText(page, foContactUsPage.subjectSelect, contactUsData.subject);
```

Comment:
You should create a method on 'contactUsPage' to select the subject and click on send button and not use them in the scenario.

Reply:
Fixed